tokens/0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7/data.json

```diff
@@ -0,0 +1,14 @@
+{
+	"contract": "0xccB32737C6dFfddFFB24CA8A96b588ac7b1822e7",
+	"type": "KRC-20",
+	"name": "WolfSafePoorPeople",
+	"symbol": "WSPP",
+	"decimals": 18,
+	"description": "WSPP This is the first currency that has a program to reduce world poverty, first launched on Binance Smart Chain, but now on Polygon, Fantom, Avalanche & KardiaChain. The establishment of the Wolfible website aims to develop the Wolf safe poor people (WSPP) project in the NFT marketplace & DeFi sector.",
+	"website": "https://wolfsafepoorpeople.com",

```

Comment:
Website is not accessible. Please update.

Reply:
Done please Check

Reply:
![Screenshot_20211118-190401_Chrome.jpg](https://user-images.githubusercontent.com/25113453/142412060-00fa56a5-70d7-4af7-be47-d4a7603d7e0d.jpg)

Still can't be access in my region. Do I need to use some kind of VPN ?

Reply:
please check my commit 2 
brother i change website to https://wolfible.com

if you want to accress wolfsafepoorpeople should use VPN

Reply:
> ![Screenshot_20211118-190401_Chrome.jpg](https://user-images.githubusercontent.com/25113453/142412060-00fa56a5-70d7-4af7-be47-d4a7603d7e0d.jpg)
> 
> Still can't be access in my region. Do I need to use some kind of VPN ?

i have change the data.json 
you can check on commit 2 
the website is change to new wolfible website brother